src/app/trainees/trainee-list/trainee-list.component.html

```diff
@@ -5,45 +5,28 @@
 </ng-container>
 
 <ng-template #notLoading>
-  <ng-container *ngIf="totalResults$ | async; else noResults">
-    <mat-table
-      *ngIf="{
+  <ng-container>
```

Comment:
were there any changes to this file? seems like you've undone the prettier formatting...?

Reply:
Ohhhh snap! My bad good spot. No totalResults come from backend so blank page running on env=local. I'll revert seems weird the table relies on results number to render though

Reply:
no probs, yea there's a few ngIf's in the html. think you also have a conflict as i also further modified this file in my pr that just went in as it also caters for the error state ui